src/platform/nrfconnect/CHIPPlatformConfig.h

```diff
@@ -69,3 +69,27 @@
 #ifndef CHIP_CONFIG_MAX_FABRICS
 #define CHIP_CONFIG_MAX_FABRICS 5
 #endif
+
+// Disable some of the too detailed log modules to save flash
+#define CHIP_CONFIG_LOG_MODULE_ExchangeManager_DETAIL 0
```

Comment:
I think we need a way to enable all logs. Can we have some `CHIP_LOG_LEVEL_SELECTIVE` Kconfig to turn this optimization on or off?

Reply:
Ok, I added CONFIG_CHIP_LOG_SIZE_OPTIMIZATION.